### PR TITLE
Ssk parsing

### PR DESCRIPTION
**Pull request by chuanqichen**

### CODE REVIEW

File: utils/dump_unique_sentences.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import csv
+import os
+# https://github.com/facebookresearch/EmpatheticDialogues/issues/21
+# https://github.com/facebookresearch/EmpatheticDialogues/issues/29
+# https://github.com/facebookresearch/EmpatheticDialogues/blob/master/empchat/datasets/empchat.py
+
+"""
+with open('empatheticdialogues/test.csv', newline='') as csvfile:
```
Comment (chuanqichen) [inline utils/dump_unique_sentences.py:8]: @kohlisimranjit Can we use the configuration parameters for the path of "test.csv"?

File: empchat/datasets/empchat.py [hunk 2]
```diff
@@ -117,9 +117,49 @@ def __init__(
         self.data = []
         self.ids = []
         history = []
+        cands = []
+        from allennlp.models.archival import load_archive
+        from allennlp.service.predictors import Predictor
+
+        #archive = load_archive("https://s3-us-west-2.amazonaws.com/allennlp/models/elmo-constituency-parser-2018.03.14.tar.gz", cuda_device=0)
+        archive = load_archive("elmo-constituency-parser-2018.03.14.tar.gz", cuda_device=0)
```
Comment (chuanqichen) [inline empchat/datasets/empchat.py:125]: @kohlisimranjit Do we need add configuration parameter to turn on/off constituency parser? This way, we can test with and without this features to do comparison test without changing code.

**Review by chuanqichen — COMMENTED** [review]
configuration parameter to turn on/off constituency parser
use the configuration for data path instead of hard coded path

**Comment by EricMichaelSmith** [comment]
Hey @chuanqichen ! Thanks for this PR :) Can you give me some context about these additions and what they're intended to do?

**Comment by EricMichaelSmith** [comment]
Closing stale PR

**Pull request closed without merging**